sdk/webpubsub/Azure.Messaging.WebPubSub/tests/Samples/WebPubSubSamples.HelloWorld.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Messaging.WebPubSub;
+using Azure.Rest.WebPubSub.Tests;
+using NUnit.Framework;
+
+namespace Azure.Template.Tests.Samples
+{
+    public class WebPubSubSamples : SamplesBase<WebPubSubTestEnvironment>
+    {
+        public void HelloWorld()
+        {
+            var endpoint = TestEnvironment.Endpoint;
+            var key = TestEnvironment.Key;
+
+            #region Snippet:WebPubSubHelloWorld
+            var client = new WebPubSubServiceClient(new Uri(endpoint), "some_hub", new AzureKeyCredential(key));
+
+            client.SendToAll("some_hub", "Hello World!");
```

Comment:
I think this is broken.
```suggestion
            client.SendToAll("Hello World!");
```